_observing-your-data/alerting/monitors.md

```diff
@@ -55,56 +54,53 @@ You query each trigger using up to 10 tags, adding the tag as a single trigger c
 
+This information is stored in plain text in the OpenSearch cluster. The encoded credentials (which are neither encrypted nor hashed) may be visible to other OpenSearch users.
 
-### Email as a destination
+### Email as a notification channel
 
-To send or receive an alert notification as an email, choose **Email** as the destination type. Next, add at least one sender and recipient. We recommend adding email groups if you want to notify more than a few people of an alert. You can configure senders and recipients using **Manage senders** and **Manage email groups**.
+To send or receive an alert notification as an email, choose **Email** as the channel type. Next, add at least one sender and recipient. Creating email groups is recommended if you want to notify several people of an alert. You can configure senders and recipients using **Email sender** and **Email recipient groups**.
 
-#### Manage senders
+#### Create sender type
 
 You need to specify an email account from which the Alerting plugin can send notifications.
 
-To configure a sender email, do the following:
+To configure a sender email:
 
-1. After you choose **Email** as the destination type, choose **Manage senders**.
-1. Choose **Add sender**, **New sender** and enter a unique name.
-1. Enter the email address, SMTP host (e.g. `smtp.gmail.com` for a Gmail account), and the port.
+1. Choose **Email** as the channel type.
+2. Choose **Sender type**, and enter sender name, and default recipients if creating a recipient group.
+1. Enter the email address, SMTP host (for example, `smtp.gmail.com` for a Gmail account), and the port.
 1. Choose an encryption method, or use the default value of **None**. However, most email providers require SSL or TLS, which require a username and password in OpenSearch keystore. Refer to [Authenticate sender account](#authenticate-sender-account) to learn more.
-1. Choose **Save** to save the configuration and create the sender. You can create a sender even before you add your credentials to the OpenSearch keystore. However, you must [authenticate each sender account](#authenticate-sender-account) before you use the destination to send your alert.
+1. Choose **Save** to save the configuration and create the sender. You can create a sender even before you add your credentials to the OpenSearch keystore. However, you must [authenticate each sender account](#authenticate-sender-account) before you use the channel to send your alert.
 
 You can reuse senders across many different destinations, but each destination only supports one sender.
```

Comment:
@AWSHurneyt Please confirm this sentence is accurate. Destinations are now notification channels, so we need to rephrase this sentence accordingly.

Reply:
@vagimeli Yes, this is accurate as of v2.0. Thank you!